source/PlayerInfo.cpp

```diff
@@ -1290,6 +1298,19 @@ bool PlayerInfo::TakeOff(UI *ui)
 			++it;
 	}
 
+	// Auto-buy commodities to fill your ships to sell along your route
+	// TODO: only with appropriate setting/mission/outfit
+	/*
+	if(!travelPlan.empty())
+	{
+		const System* destination = travelPlan.front();
+		//TODO: Find system within your jump range?
+
+		BuyBestTrade(*destination);
+	}
+	*/
+
+
```

Comment:
```suggestion
```
Seems you forgot to clean this up, if this PR was a draft it wouldn't matter but it isn't.
Maybe it is linked to another PR and waiting for it or smth? Idk if it should stay here anyway

Reply:
This PR being a draft now, this is where automated trades will go in